### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Kwamay**
In this module I...........

-[ ] Fetch an API data from a NewApp to my project

-[ ] Run a unit test using react library for the component created

-[ ] Styled the project per the project requirement

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,182 @@
...
+- **Displays the estimated headlines of the news on the homepage, also directly from the API.**
+- **Displays headline news with description, title, author and date published**
+- **Uses a mobile web responsive layout format that looks good and are easy to use on mobile devices.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo]()
```
Comment (Alusp) [inline README.md]: - [ ] Good job so far, :clap: , However, Please ensure that you add a live demo version of your App, currently you do not have it. :slightly_smiling_face:

File: README.md [hunk 2]
```diff
@@ -1 +1,182 @@
...
+- [Video Presentation](https://www.loom.com/share/03dde572f79041519a2f15be5b913f06?sid=36d18937-ac9b-451d-bc27-f567d3df258d)
```
Comment (Alusp) [inline README.md]: - [ ] Please ensure that you mention a piece of code you are most proud of in your video presentation. :slightly_smiling_face:

File: src/component/Nav.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import '../CSS/nav.css';
+import { GiNewspaper } from 'react-icons/gi';
+import { IoSettingsOutline } from 'react-icons/io5';
+import { MdOutlineArrowBackIosNew } from 'react-icons/md';
+
+const Nav = () => (
+  <nav className="nav-container">
+    <Link className="nav-back" to="/">
+      <MdOutlineArrowBackIosNew />
+    </Link>
```
Comment (Alusp) [inline src/component/Nav.jsx]: - [ ] According to the project requirement, the back button is supposed to be displayed only on the details page and not on the Home page, Please resolve that. :slightly_smiling_face: 
- backbtnOnHome

File: src/component/Details.jsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  // Find the selected news item in the filtered news
+  const selectedNews = news.find((item) => item.source.name === id);
+
+  return (
+    <div>
+      {selectedNews && (
+        <ul className="details-container">
+          <li className="details-content">
+            <img className="details-img" src={selectedNews.urlToImage} alt="" />
+          </li>
```
Comment (Alusp) [inline src/component/Details.jsx:26]: - [x] Your details page images are stretched out, which destroys the look and feel of your design, Please ensure that you style them accordingly. 
- DetailsStretched

- [x] Please ensure that you use the `lato` and `gill sans` fonts family given in the design requirement.
- [x] And please make sure you Layout: composition and space between elements in your details page.

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @Kwamay, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
close

## Highlights

- [ ] Use Professional Readme file 👍
- [ ] Descriptive PR Summary used 👍
- [ ]  Use React components.👍
- [ ] Handle events in a React app. 👍
- [ ] Use store, actions, and reducers in React. 👍


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ademibowale — APPROVED** [review]
Hi @Kwamay  , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️
- Use functional components✔️
- follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))✔️
- The project contains a `Home` page and a `Detail` page✔️
- The category item detail page has its unique route within the SPA.
- Descriptive commit messages ✔️ 
- App has unit tests for several components of the app
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**